l10n_br_sale/models/sale_order_line.py

```diff
@@ -122,19 +122,17 @@ def _onchange_product_uom(self):
     def _onchange_discount_percent(self):
         """Update discount value"""
         if not self.env.user.has_group('l10n_br_sale.group_discount_per_value'):
-            if self.discount:
-                self.discount_value = (
-                    (self.product_uom_qty * self.price_unit)
-                    * (self.discount / 100)
-                )
+            self.discount_value = (
```

Comment:
Não precisa verificar se tem desconto ?

Reply:
No, the default is zero, and in this case the result will still be consistent.